model.py

```diff
@@ -3480,29 +3484,50 @@ def calculate_presentation(self, policy=None, search_index_client=None):
                 # edition is still an option, choose it.
                 new_primary_edition = potential_primary_edition
 
-        if new_primary_edition:
-            self.primary_edition = new_primary_edition
+        # Note: policy.choose_edition is true in default PresentationCalculationPolicy.
+        # If we don't have a self.primary_edition by now, this will just set all the editions' 
+        # is_primary_for_work attributes to False.
```

Comment:
I don't think the second part of this comment is true anymore - what should happen if we don't have a primary edition by now?